app/frontend/src/cljs/teet/contract/contract_partners_view.cljs

```diff
@@ -175,6 +237,73 @@
        [foreign-fields]
        [estonian-form-section business-search-failed? exception-in-xroad?])]))
 
+(defn- last-modified [selected-company]
+  (str (tr [:common :last-modified]) " "
+    (if (nil? (:meta/modified-at selected-company))
+      (str (format/date-time (:meta/created-at selected-company)) " "
+        (get-in selected-company [:company-contract/company :meta/creator :user/given-name])
+        " "
+        (get-in selected-company [:company-contract/company :meta/creator :user/family-name]))
+      (str (format/date-time (:meta/modified-at selected-company)) " "
+        (get-in selected-company [:company-contract/company :meta/modifier :user/given-name])
+        " "
+        (get-in selected-company [:company-contract/company :meta/modifier :user/family-name])))))
```

Comment:
We have in the user-model ns a function called user-name, which could be used here.